lec06-pop-models.Rmd

```diff
@@ -2,3 +2,298 @@
 title: "Qualitative review of population models"
 ---
 
+
+## Lesson preamble
+
+In developing this series of lectures I have referred heavily to the notes from Nonlinear Physics,
+[PHY1460](https://www.physics.utoronto.ca/students/graduate-courses/current/phy1460s), 
+which I would encourage anyone who is interested in getting more in depth on the quantitative analysis 
+of dynamical systems to take or audit. I also refer to "Nonlinear Dynamics and Chaos" by Steven Strogatz, 
+a very good textbook which you might find helpful. You can find the pdf online, it's available as an e-book
+from the UofT library, and it's available in the Physics library as a hard copy course reserve.
+
+## Learning objectives:
+
+- Appreciate the purpose and usefulness of modelling real-world systems
+- Understand the use of differential equations to model populations
+- Qualitatively analyze one-dimensional population models by hand and in R
+    - Find fixed points graphically
```

Comment:
Graphically meaning by hand or graphically meaning in R?

Reply:
Both, I guess? They make the graphs in R and then find the fixed points by looking at it.